guides/deployments/postgres.md

```diff
@@ -8,13 +8,18 @@ database.
 
 ## Background
 
-By default, Coder deploys a [TimescaleDB](https://www.timescale.com) inside the
-Kubernetes cluster to which you've installed Coder. However, we recommend this
-**only for evaluation purposes**, since the database isn't backed up and can be
-lost if the cluster goes down.
-
-As such, we strongly recommend using a PostgreSQL database for production
-deployments and hosting it **outside** the Kubernetes cluster hosting Coder.
+For convenience and ease of installation, Coder's default Helm Chart settings
+will  [PostgreSQL database](https://www.postgresql.org/) within the
+installation's Kubernetes namespace. This is useful for evaluation purposes;
+however, we **recommend using an out-of-cluster database for production**, to
+streamline maintenance operations, such as backups and upgrades. The database
+container is not backed up and will be lost when deleting the namespace or
+Kubernetes cluster.
+
+> For optimal performance, it is important to ensure that the round-trip latency
+> between the Coder control plane services and the database is low. We recommend
+> ensuring that the database is within the same datacenter as the control plane,
+> such as within the same cloud availability zone.
```

Comment:
```suggestion
> or within the same availability zone within a cloud provider.
```

Reply:
I used this phrasing to distinguish between deploying within the same region, but not the same AZ. I think "same availability zone" is understood to mean "within the same cloud provider", no?

Most of the clouds use "availability zone" to refer to a specific failure domain (e.g. power supplies, building, etc) and we would typically consider this to be the same data center. There are sometimes multiple availability zones that are different buildings in close proximity within a region, so latency is still low in that case.

I think as long as users can guarantee low latency, it doesn't matter where the database is, so we shouldn't be prescriptive that this _must_ be in the same AZ.